src/app/authors/authors.component.html

```diff
@@ -10,7 +10,7 @@ <h2>Current Contributors</h2>
 <div class="row mt-5 mb-5">
     <div class="col-sm float-right">
         <p>Goblint is developed at the Technische Universität München's Chair of Formal Languages, Compiler Construction, Software
-            Construction (<a href="http://www2.in.tum.de">Helmut Seidl</a>) and University of Tartu's Laboratory for Software Science (<a href="http://sws.cs.ut.ee">Vesal Vojdani</a>). It is distributed under the MIT license. </p>
+            Construction (<a href="https://www.cs.cit.tum.de/en/pl/home/">Helmut Seidl</a>) and University of Tartu's Laboratory for Software Science (<a href="http://sws.cs.ut.ee">Vesal Vojdani</a>). It is distributed under the MIT license. </p>
```

Comment:
Wouldn't hurt to also use HTTPS for ours:
```suggestion
            Construction (<a href="https://www.cs.cit.tum.de/en/pl/home/">Helmut Seidl</a>) and University of Tartu's Laboratory for Software Science (<a href="https://sws.cs.ut.ee">Vesal Vojdani</a>). It is distributed under the MIT license. </p>
```